tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/TelemetryMetricGenerator/CiVisibilityMetricsTelemetryCollector_CountCIVisibility.g.cs

```diff
@@ -341,7 +390,7 @@ private static AggregatedMetric[] GetCountCIVisibilityBuffer()
     /// It is equal to the cardinality of the tag combinations (or 1 if there are no tags)
     /// </summary>
     private static int[] CountCIVisibilityEntryCounts { get; }
-        = new int[]{ 40, 80, 10, 10, 4, 1, 2, 10, 2, 9, 63, 1, 5, 1, 5, 1, 5, 8, 1, 5, 1, 1, 4, 4, 4, 1, 1, 1, 5, };
+        = new int[]{ 40, 80, 10, 10, 4, 1, 4, 22, 2, 9, 63, 2, 11, 2, 11, 2, 11, 8, 2, 11, 1, 1, 4, 4, 4, 1, 1, 2, 11, };
```

Comment:
This looks rather error-prone, I'd consider calculating these indices automatically if they aren't yet

Reply:
This file is auto-generated 🙂